apps/platform/trpc/routers/orgRouter/iCanHaz/iCanHazRouter.ts

```diff
@@ -0,0 +1,22 @@
+import { z } from 'zod';
+import { router, orgProcedure, createCallerFactory } from '~platform/trpc/trpc';
+import { billingTrpcClient } from '~platform/utils/tRPCServerClients';
+
+export const iCanHazRouter = router({
+  domain: orgProcedure.input(z.object({})).query(async ({ ctx }) => {
```

Comment:
you can drop the `.input(z.object({}))` part, then the query would not need to pass `{}` while calling the endpoint. It would make the endpoint do 1 less check